shared/cluster.go

```diff
@@ -584,3 +584,20 @@ func processNodeArgs(nodeArgs string) (nodeArgsMapSlice []map[string]string) {
 
 	return nodeArgsMapSlice
 }
+
+func CreateSecret(secret, namespace string) error {
+	kubectl := fmt.Sprintf("kubectl --kubeconfig %s", KubeConfigFile)
+	if namespace == "" {
+		namespace = "default"
+	}
+	cmd := fmt.Sprintf("%s create secret generic %s -n %s --from-literal=mykey=mydata",
+		kubectl, secret, namespace)
+	stdout, err := RunCommandHost(cmd)
```

Comment:
same comment as  
"

is there any other more meaningful name to this var instead of stdout ? we could be more specific here
"

Reply:
fixed